### DISCUSSION TITLE
Don`t know what i am doing wrong

### DISCUSSION THREAD

**Discussion by Araxion** [fuatakgun/eufy_security#1176] [Q&A]
Hello friend,
i am trying integrate E330 Professional Cam. I successfully installed and conifgured eufy-security-ws
<img width="1060" alt="Snímek obrazovky 2024-07-15 v 8 59 08" src="https://github.com/user-attachments/assets/f1e436d7-ad0f-4af0-bb7b-f254b147fabe">
than i installed and connected go2rtc server
<img width="1006" alt="Snímek obrazovky 2024-07-15 v 8 59 18" src="https://github.com/user-attachments/assets/3644855a-d5a9-4c4b-995c-41eb0c47e899">
<img width="578" alt="Snímek obrazovky 2024-07-15 v 8 59 37" src="https://github.com/user-attachments/assets/6ff1e56a-b11f-4bd6-8d48-6eb308c44850">
<img width="606" alt="Snímek obrazovky 2024-07-15 v 9 00 00" src="https://github.com/user-attachments/assets/2bbadd69-1159-494a-8f80-7e0cf382b2bb">
But i still can`t get live footage from camera. All i get is static image of last event. Video quality is set to 720p (lowest possible), and in app i am using another admin account than in integration.

Any help please? Thanks!

**Comment by Jogs62** [comment]
Hi, I have a Eufy E340 and same issue. I have the last event image but impossible to have the live video. Did you find a solution ?
